scss/_patterns_footer.scss

```diff
@@ -6,6 +6,7 @@
     border-top: 1px solid $color-mid-light;
     font-size: .875rem;
     line-height: 1.5;
+    margin-top: auto;
```

Comment:
This looks like a good use case for `@root`. Root allows you to add styling to parent elements from within a pattern. This means if someone doesn't include the footer pattern then it wont apply the height to the body.

https://www.sassmeister.com/gist/5cd04a820b838320f16a9ce29254e103

Reply:
@anthonydillon I didn't know this was a thing. Thanks!